extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js

```diff
@@ -934,7 +904,8 @@ export class AmpLightboxViewer extends AMP.BaseElement {
       this.topBar_.classList.remove('fullscreen');
     }
     toggle(dev().assertElement(this.carousel_), true);
-    this.toggleDescriptionBox_(true);
+    this.updateDescriptionBox_();
```

Comment:
Does `this.updatedesciptionBox_` need to be called when `this.descriptionBox_` is toggled? It could be good defensive practice to create one method that calls both of them so no one risks forgetting to call `this.updatedesciptionBox_` when toggling it.

Reply:
True. I'll bring that function back.